adhoc/blosc_memleak_check.py

```diff
@@ -1,13 +1,13 @@
 import sys
 
-import numcodecs
 import numpy as np
 from numpy.testing import assert_array_equal
 
+import numcodecs
 
 codec = numcodecs.Blosc()
 data = np.arange(int(sys.argv[1]))
-for i in range(int(sys.argv[2])):
+for _i in range(int(sys.argv[2])):
```

Comment:
```suggestion
for _ in range(int(sys.argv[2])):
```
Any reason not to just go with a single underscore?

Reply:
No, I was trying to stay as close as possible to the original code, but in this case it doesn't make sense. Changed to `_`.